spec/boolean_spec.rb

```diff
@@ -225,7 +225,8 @@
         end
         it 'should have results that match first term but not second term' do
           # titles 'Indochine'
-          indochine_results = %w(383033 430603 4312384 3083716 3065221 2134301)
+          indochine_results = %w(383033 430603 1921469 3083716 3065221 2134301 10316540)
+          # 4312384 withdrawn
```

Comment:
ditto on this comment - another way to deal with this is to make an individual commit with this in the commit message (maybe you did - I haven't looked at individual commits yet)